app/bundles/EmailBundle/EventListener/ReportSubscriber.php

```diff
@@ -406,7 +406,11 @@ public function onReportGraphGenerate(ReportGraphEvent $event)
     {
         $graphs = $event->getRequestedGraphs();
 
-        if (!$event->checkContext(self::CONTEXT_EMAIL_STATS) || ($event->checkContext(self::CONTEXT_EMAILS) && !in_array('mautic.email.graph.pie.read.ingored.unsubscribed.bounced', $graphs))) {
+        if (!$event->checkContext([self::CONTEXT_EMAIL_STATS, self::CONTEXT_EMAILS])) {
```

Comment:
Why was `mautic.email.graph.pie.read.ingored.unsubscribed.bounced` excluded to start with?

Reply:
It's the only graph on the `self::CONTEXT_EMAILS` context so the intent was to continue with the execution only in that case. But the part before `||` was true in this case so it was never executing for that graph.